src/java.desktop/macosx/native/libawt_lwawt/awt/JavaComponentAccessibility.m

```diff
@@ -286,39 +283,6 @@ + (void)initialize
     if (sRoles == nil) {
         initializeRoles();
```

Comment:
Do we need to delete the JavaAccessibilityIgnore roles from the "sRoles"?

Reply:
Not right now, there is a legacy code that still uses it and until i am ready to remove all of it i want to keep everything functioning - if anything for the comparison stakes. In the end the entire sRoles map will be eliminated.